.github/workflows/update-font.yml

```diff
@@ -0,0 +1,38 @@
+name: Font update check
+
+# **What it does**: Checks that the font cachebuster parameter has been updated if the font file has changed
+# **Why we have it**: To make sure that browser font caching is invalidated and the updated font is used
+# **Who does it impact**: Any PR which changes the font file
+
+on:
+  pull_request_target:
+    types: [opened, reopened]
+    paths:
+      - 'plugins/Morpheus/fonts/matomo.woff2'
+
+permissions:
+  actions: read
+  checks: read
+  contents: read
+  deployments: none
+  issues: read
+  packages: none
+  pull-requests: read
+  repository-projects: none
+  security-events: none
+  statuses: read
+
+jobs:
+  check:
+    runs-on: "ubuntu-latest"
+    steps:
+     - uses: actions/checkout@v3
+     - name: Compare stylesheet to base branch
+       run: |
+         if [[ $(wget -O - -o /dev/null https://raw.githubusercontent.com/matomo-org/matomo/$GITHUB_BASE_REF/plugins/Morpheus/stylesheets/base/icons.css | sed '4q;d') == $(sed '4q;d' plugins/Morpheus/stylesheets/base/icons.css) ]]
```

Comment:
If I understand the bash correct this will simply compare the 4th line of the file, correct? This could possibly fail in the future if someone decides to add a line upfront. 
Should we maybe add a comment to the CSS file to ensure this won't happen by accident? Or would it be better to look for the line that e.g. contains `format('woff2')`

Reply:
That makes sense, using a specific line number makes the check fragile :+1: I've replaced checking the exact line number using sed with grep instead, so the first line containing `format('woff2') will be used for comparison wherever it occurs in the file.